tests/samples/simple_pack_pipeline_e2e.mlir

```diff
@@ -1,4 +1,4 @@
-// RUN: iree-compile --iree-hal-target-backends=amd-aie --compile-to=executable-sources %s | iree-opt --pass-pipeline="builtin.module(hal.executable(hal.executable.variant(iree-hal-translate-target-executable-variants{target=amd-aie})))" | FileCheck %s --check-prefix=CPP
+// RUN: iree-compile --split-input-file --iree-hal-target-backends=amd-aie --compile-to=executable-sources %s | iree-opt --pass-pipeline="builtin.module(hal.executable(hal.executable.variant(iree-hal-translate-target-executable-variants{target=amd-aie})))" | FileCheck %s --check-prefix=CPP
```

Comment:
I wonder if giving `--split-input-file` to the iree-opt command as well will solve the issue for you, not sure how the piping the output of iree-compile to it interacts with that but pretty sure that ahs something to do with the module having both fucntions

Reply:
I did try adding `--split-input-file` after the pipe to `iree-opt` as well. :P

It still behaved the same way. I can give it a fresh try tomorrow - but beside that I'm out of options unless I need to look into the AIETarget/AIELowerExecutable (highly unlikely but that would be the only place to check).

It's strange though since the individual tests pass - just while putting them in the same file it misbehaves.

Reply:
Is this problem solved? `--split-input-file` seems to work fine when I added it in this test https://github.com/nod-ai/iree-amd-aie/blob/main/tests/samples/simple_pack_pipeline_e2e.mlir#L1

Reply:
Hi @yzhang93 , yes - it seems to have been resolved. I've removed the separate lit tests.

Reply:
Good to hear this has been resolved! I had a closer look at how to use different test inputs and commands in one file, in case you haven't seen that I am using it in my PR https://github.com/nod-ai/iree-amd-aie/pull/159